www/README.md

```diff
@@ -1,16 +1,51 @@
-# React documentation site
-
-To get setup:
-```
-yarn install
-```
-
-To run the dev server:
-```
-yarn run dev
-```
-
-To clear the local cache:
-```
-yarn run reset
-```
\ No newline at end of file
+# React — documentation site
+
+## Getting started
+
+### Prerequisites
+
+1. Git
+1. Node: install version 7.6 or greater
+1. Yarn: `npm i -g yarn` to install it globally via NPM
+1. A clone of the [https://github.com/facebook/react](React repo) on your local machine
+1. A fork of the repo (for any contributions)
+
+### Installation
+
+1. `cd www` to go into the website project
+1. `yarn` to install the website's NPM dependencies
+
+### Running locally
+
+1. `yarn dev` to build the site with Gatsby, a (React-based!) static site generator
```

Comment:
perhaps change to "to start the hot-reloading development server powered by Gatsby"?

Reply:
Sure, and drop the ` a (React-based!) static site generator`?